### PR TITLE
Concat + minify

### LINKED ISSUE #51: Concat and minify our files
Possibly roll into the showcase file too?

### PR DESCRIPTION
**Pull request by WPFerg**
Fixes #51

### CODE REVIEW

File: gruntfile.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ module.exports = function(grunt) {
             js: {
                 expand: true,
                 cwd: 'src/',
-                src: ['**/*.js', '**/*.json'],
+                src: ['**/*.json'],
                 dest: 'public'
             },
```
Comment (owennw) [inline gruntfile.js]: It feels weird to me that the only `src` files in the `js` object are `json`..

File: gruntfile.js [hunk 2]
```diff
@@ -216,10 +232,11 @@ module.exports = function(grunt) {
...
 
-    grunt.registerTask('build', ['eslint', 'clean', 'showcase', 'copy', 'less:development', 'connect:livereload']);
+    grunt.registerTask('build', ['eslint', 'clean', 'showcase', 'copy', 'concat:dist', 'less:development', 'connect:livereload']);
+    grunt.registerTask('build:uglify', ['eslint', 'clean', 'showcase', 'copy', 'uglify', 'less:development', 'connect:livereload']);
     grunt.registerTask('serve', ['build', 'openfin:serve']);
-    grunt.registerTask('createZip', ['build', 'download']);
-    grunt.registerTask('ci', ['build', 'download']);
+    grunt.registerTask('createZip', ['build:uglify', 'download']);
+    grunt.registerTask('ci', ['build:uglify', 'download']);
     grunt.registerTask('deploy', ['ci', 'gh-pages:origin']);
```
Comment (owennw) [inline gruntfile.js]: So `concat` is only run for dev, and `uglify` run for prod, as it concats the files itself?

Comment (WPFerg) [inline gruntfile.js]: Yeah. You can do both, I believe

File: gruntfile.js (hunk 1 above)
Comment (WPFerg) [inline gruntfile.js]: I'll rename it to `json`.

**Comment by WPFerg** [comment]
Pushed up some changes, I've streamlined the task registration so `build:uglify` merely runs `build`, then `uglify`.

**Pull request merged**